presto-main/src/main/resources/webapp/vendor/bootstrap/config.json

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
Also, are you sure all the necessary license files are included? It seems like the license for Solarized, at least, is missing: https://github.com/altercation/solarized/blob/master/LICENSE